coprarna_aux/CopraRNA2html.r

```diff
@@ -0,0 +1,294 @@
+	cons<-paste0("[Phylogenetic target conservation](",cons1,"){target='_blank'}")
+	cons2<-paste0("![Phylogenetic target conservation](",cons1,"){width=100% height=600}")
+	ma<-c(ma,paste("## Overview {.tabset .tabset-fade .tabset-pills}","### Phylogenetic target conservation",cons,"\n",cons2,"\n","### Functional enrichment",enrich,"\n",enrich2,"\n","### mRNA regions plota",mrna_reg,"\n",mrna_reg2,"\n","### sRNA regions plots",srna_reg,"\n",srna_reg2,"\n","### Auxiliary enrichment","\n",sep="\n"))
+	
+	prefix<-paste("```{r,echo=F}","wd<-getwd()",sep="\n")
+	suffix<-paste("kable((tab)) %>% ","kable_styling(c('striped', 'bordered')) %>% ","kable_styling(fixed_thead = T) %>% ","kable_styling(full_width = F)",  "```",sep="\n")
+	aux2<-c("tab<-read.csv(",aux,",sep=',')")
+	
+	ma<-c(ma,prefix,aux2,suffix)
+	ints<-paste("./evo_alignments2/",selection,"/interactions.html",sep="")
+	ints<- paste0("[interaction](",ints,"){target='_blank'}")
+	#d<-dir("./jalview")
+	#mrna<-paste("./jalview/",d[match(paste(selection,"_mRNA.PNG",sep=""),sub(".*?_","",d))],sep="")
+	#mrna<-paste0("[mRNA_alignment](",mrna,"){target='_blank'}")
+	#srna<-paste("./jalview/",d[match(paste(selection,"_sRNA.PNG",sep=""),sub(".*?_","",d))],sep="")
+	#srna<-paste0("[sRNA_alignment](",srna,"){target='_blank'}")
+	heat<-paste("./evo_alignments2/",selection,"/",selection,"_conservation_heatmap.pdf",sep="")
+	#heat<-paste("./heatmaps/",selection,".pdf",sep="")
+	heat<- paste0("[conservation](",heat,"){target='_blank'}")
+	probcons<-paste("./evo_alignments2/",selection,"/spot_probabilities.png",sep="")
+	probcons<- paste0("[conserved_peaks](",probcons,"){target='_blank'}")
+	#Links<-paste(heat,mrna,srna,ints, sep=", ")
+	Links<-paste(heat,ints,probcons, sep=", ")
+	ind_tables<-paste(getwd(),"/evo_alignments2/ind_tables",sep="")
+	dir.create(ind_tables)
+	
+	
+	tab<-cbind(tabs[[1]], Links)
+	write.table(tab, file=paste(ind_tables,"/",names(tabs)[1],".txt",sep=""), sep="\t", row.names=FALSE, quote=F)
+	tmp<-paste("'./evo_alignments2/ind_tables/",names(tabs)[1],".txt'",sep="")
+	tmp<-c("tab<-read.csv(",tmp,",sep='\\t')")
+	ma<-c(ma,"\n", paste("## Prediction organisms of interest",paste("### ", nam2[1],sep=""),sep="\n"),prefix,tmp,suffix, "## Predictions other organisms {.tabset .tabset-fade .tabset-pills}")
+	
+	if(length(tabs)>1){
+		for(i in 2:length(tabs)){
+			tab<-cbind(tabs[[i]], Links)
+			write.table(tab, file=paste(ind_tables,"/",names(tabs)[i],".txt",sep=""), sep="\t", row.names=FALSE, quote=F)
+			tmp<-paste("'./evo_alignments2/ind_tables/",names(tabs)[i],".txt'",sep="")
+			tmp<-c("tab<-read.csv(",tmp,",sep='\\t')")
+			ma<-c(ma,"\n", paste("### ", nam2[i],sep=""),prefix,tmp,suffix)
+		}
+	}
+	
+	writeLines(ma, con = "markdown_final.Rmd", sep = "\n", useBytes = FALSE)
+	rmarkdown::render(paste(getwd(),"/markdown_final.Rmd",sep=""),output_file='CopraRNA2_result.html',intermediates_dir=getwd(),knit_root_dir=getwd(),output_dir=getwd(),clean =F)

```

Comment:
`getwd()` wird nicht immer funktionieren... da sollte auch der CopraRNAauxPath hin..

Reply:
Das soll ja gerade nicht im CopraRNAauxPath Path ausgeführt werden, sondern in dem CopraRNA results folder